docs_user/modules/proc_migrating-object-storage-data-to-rhoso-nodes.adoc

```diff
@@ -17,7 +17,7 @@ The {object_storage_first_ref} migration involves the following steps:
 * For DNS servers, ensure that all existing nodes are able to resolve the hostnames of the {rhocp_long} pods, for example, by using the external IP of the DNSMasq service as the nameserver in `/etc/resolv.conf`:
 +
 ----
-$ oc get service dnsmasq-dns -o jsonpath="{.status.loadBalancer.ingress[0].ip}" | CONTROLLER1_SSH tee /etc/resolv.conf
+$ oc get service dnsmasq-dns -o jsonpath="{.status.loadBalancer.ingress[0].ip}" | $CONTROLLER1_SSH sudo tee /etc/resolv.conf
```

Comment:
is this "sudo" really needed here?

Reply:
It is needed in cases when non-sudo user is used to perform adoption i.e, ssh to controllers.
[stack@undercloud-0 ~]$ oc get service dnsmasq-dns -o jsonpath="{.status.loadBalancer.ingress[0].ip}" | $CONTROLLER1_SSH tee /etc/resolv.conf
192.168.23.80tee: /etc/resolv.conf: Permission denied